.gitignore

```diff
@@ -22,6 +22,9 @@ DerivedData
 *.xcuserstate
 ios/.xcode.env.local
 
+# This is temporary until we have an app logo
+ios/BlossomsCam/Images.xcassets
+
```

Comment:
This is a pretty cool thing to think about. Having a temporary image in the meantime is a good idea.